question-bank.json

```diff
@@ -195,13 +166,133 @@
+            "Sunlight exerts a force of 1 micronewton per square meter. Ignoring friction, how long would it take a car powered by this force to go from 0 mph to 60 mph (or, equivalently, from 0 kph to 100 kph)?": 11,
+            "How many babies are born in New Haven on an average day?": 1,
+            "How many trees are there in Connecticut?": 7,
+            "Despite what you may have learned in chemistry class, any pair of atoms will stick to each other to form a diatomic molecule. Given this fact, how many species of diatomic molecules are there?": 2,
+            "You have a one-­‐liter bottle of water. What is the area you could cover with this water if it were spread to be one molecule thick?": 7,
+            "How long a line can an average pencil make?": 3,
+            "If all the soda that is drunk in the United States during a typical day were gathered into a single drop, what would the drop’s radius be?": 2,
+            "How many generations back do you have to go until your total number of ancestors is equal to the world population?": 1,
+            "If electric energy costs $0.1 per kilowatt hour, and is delivered to your home at a voltage of 120 Volts, what is the cost of one electron? ": -24
+        },
+        "njaapt": {
+            "How many piano tuners are there in Chicago?": 2,
+            "What is pi to the power of 9?": 4,
+            "How many pounds of rice were consumed by people living in the U.S. last year?": 10,
+            "How many board game dice does it take to equal the mass of the average human?": 4,
+            "What number of tennis balls would you have to lay on the surface area of an Olympic-sized pool in order to fully cover it?": 5,
+            "How much trash (in pounds) does the average family produce in a year?": 3,
+            "What volume of air (in Liters) do you breathe in one day?": 3,
+            "How many hairs are on the average human head (that actually has hair!)": 5,
+            "If we could brew coffee in swimming pools, how many pools would we need to satisfy the need in the U.S. on a given day?": 2,
+            "How many pennies would it take to make a stack the height of the Empire State Building?": 5
+        },
+		"haadi": {
+			"How many grams of cheese does Wisconsin produce every day?": 9
+		}
     },
     "sources": {
         "og": {
             "link": "https://github.com/EricAndrechek/FermiQuestions#changes",
             "author": "Eric Andrechek",
             "date": "February 10, 2019"
+        },
+        "nc_state_ex": {
+            "link": "https://ncscienceolympiad.ncsu.edu/wp-content/uploads/2018/08/NC-FERMI-C.pdf",
+            "author": "NC State",
+            "date": "07/29/22"
+        },
+        "nc_state": {
+            "link": "https://ncscienceolympiad.ncsu.edu/wp-content/uploads/2018/08/2018-Fermi-Questions-State-C.pdf",
+            "author": "NC State",
+            "date": "07/29/22"
```

Comment:
Also, do you think we should change the dates on these? I know they were originally sourced in 2019, do you think we leave the date as that when originally sourced or update it to now? I don't really think it matters, I just figure we either change the date back to 2019 and put me as the author of the addition or leave the date as now and put you as the author of the addition. Your call.